doc/fitting.rst

```diff
@@ -15,59 +15,17 @@ when using complicated constraints or comparing results from related fits.
 
 
 The :func:`minimize` function
-===============================
+=============================
+
+.. currentmodule:: lmfit.minimizer
```

Comment:
With 

```
..currentmodule:: lmfit.minimizer
```

sphinx creates a link to a function using the intuitive syntax:

```
:func:`minimize`
```

When I change the directive to 

```
..currentmodule:: minimizer
```

the link is not created anymore. Is there a different syntax in this case?

In general, I would favor cleaner/shorter internal links over shorter intersphinx links.